src/opensearch-dashboards/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Use OpenSearch image as base
+FROM opensearchproject/opensearch-dashboards:2.7.0
```

Comment:
can this be 2.8.0?

Reply:
this should corresponds to the OpenSearch server...